examples/connection.rb

```diff
@@ -0,0 +1,27 @@
+require "yast"
+require "y2network/config"
+require "y2network/virtual_interface"
+require "y2network/connection_config/ethernet"
+require "y2network/connection_config/bond"
+require "y2network/connection_config/wireless"
+
+# given an interfaces list (interfaces)
+
+config = Y2Network::Config.from(:sysconfig)
```

Comment:
to be honest I still do not like this initialization. Ideally it should be `Y2Network::Config.read` and it will read configuration whatever it want and however. I am not interested if it use sysconfig, wicked xml or network manager udev service. Just read current configuration.

Reply:
It is fine for me to have such a method, but you need to detect which backend you are using. So, for me, it makes sense to have a `Y2Network::Config.read` which finds out which is the backend and then it just simply call to `Config.from` with the proper value.

Reply:
yes Config.from makes sense for me, but in general I think Config.read should be preferred way to read configuration.